test-utils/fixtures/messages/BaseMessages.scala

```diff
@@ -38,6 +38,7 @@ trait BaseMessages { _: i18n =>
   def lrnSubheading(lrn: String): String = s"Create movement for $lrn"
   val continue = "Continue"
   val confirmAnswers = "Confirm answers"
+  val confirmCodes = "Confirm codes"
```

Comment:
As in messages, this seems to be specific to commodity codes? So is it worth moving to ConfirmCommodityCodeMessages/ItemConfirmCommodityCodeMessages